src/io/flutter/jxbrowser/EmbeddedBrowser.java

```diff
@@ -5,25 +5,29 @@
  */
 package io.flutter.jxbrowser;
 
+import com.intellij.openapi.util.SystemInfo;
 import com.intellij.openapi.wm.ToolWindow;
 import com.intellij.ui.content.Content;
 import com.intellij.ui.content.ContentManager;
 import com.teamdev.jxbrowser.browser.Browser;
 import com.teamdev.jxbrowser.engine.Engine;
 import com.teamdev.jxbrowser.engine.EngineOptions;
+import com.teamdev.jxbrowser.engine.RenderingMode;
 import com.teamdev.jxbrowser.view.swing.BrowserView;
 import icons.FlutterIcons;
 
 import java.awt.*;
 
 import static com.teamdev.jxbrowser.engine.RenderingMode.HARDWARE_ACCELERATED;
+import static com.teamdev.jxbrowser.engine.RenderingMode.OFF_SCREEN;
 
 public class EmbeddedBrowser {
   public void openPanel(ContentManager contentManager, String tabName, String url) {
-    EngineOptions options =
-      EngineOptions.newBuilder(HARDWARE_ACCELERATED).build();
-    Engine engine = Engine.newInstance(options);
-    Browser browser = engine.newBrowser();
+    final RenderingMode renderingMode = SystemInfo.isMac ? OFF_SCREEN : HARDWARE_ACCELERATED;
```

Comment:
Add a TODO with a bug # to switch back to HARDWARE_ACCELERATED rendering once the JxBrowser bug is fixed upstream.
Also, how is the frame rate with OFF_SCREEN rendering? DevTools uses WebGL heavily so there is a risk OFF_SCREEN rendering is too slow to enable as the default on Mac.

Reply:
The OFF_SCREEN frame rate is estimated at 23 FPS, compared to 60 FPS for HARDWARE_ACCELERATED. Is there a frame rate that WebGL requires?